example/src/components/PhoneInputNumber/index.tsx

```diff
@@ -46,7 +46,8 @@ export const PhoneInputNumber = (props: PhoneInputParams) => {
   }
 
   useEffect(() => {
-    if((defaultValue && userphoneNumber) || defaultValue === undefined || defaultValue === ''){
+    if((defaultValue && userphoneNumber) || defaultValue === undefined || defaultValue === '' || defaultValue === null){
```

Comment:
the validation `!defaultValue` is equal to `defaultValue === undefined || defaultValue === '' || defaultValue === null`, please improve this part